style: initial styling

ljkbennett opened the pull request.

Commit:
style: update reactions to use modular css

Commit:
style: initial styling (page layout, headings)

Commit:
chore: remove obsolete test

littleowllabs commented:
Ebert has finished reviewing this Pull Request and has found:

* 3 possible new issues (including those that may have been commented here).

You can see more details about this review at https://ebertapp.io/github/ItsJuice/juicebox/pulls/67.

<!-- Ebert auto generated comment -->

Commit:
chore: put svg-react-loader in right place

Commit:
fix: fix sockets

Commit:
fix: make page connect to stream on load

Previously used to use redux-router which would always cause a store change on
load. Since we switched to react-redux-router this is no longer the case so we
need to explicitly connect to gain initial state.

Commit:
chore: remove debug

ljkbennett commented:
Ebert has finished reviewing this Pull Request and has found:

* 3 possible new issues (including those that may have been commented here).
* 2 fixed issues! 🎉

You can see more details about this review at https://ebertapp.io/github/ItsJuice/juicebox/pulls/67.

<!-- Ebert auto generated comment -->

OCannings committed:
fix(Reactions): remove stutter when looping

The reaction videos would occasionally stutter when their videos looped.
This was caused by the video becoming temporarily "empty" or transparent
at the point of looping.

This has been fixed by checking if the video frame is blank before
rendering the frame.

OCannings committed:
Merge pull request #68 from ItsJuice/fix/frame-animation-stutter

fix(Reactions): remove stutter when looping

OCannings committed:
feat(Youtube): remove category scope

Searches are no longer limited to the music category!

OCannings committed:
style(Youtube): add thumbnails to results

Thumbnails have been added to the search results to help differentiate
between similar videos.

OCannings committed:
refactor(Search): add scss file for results styles

To avoid passing the styles down from the search component.

OCannings committed:
Merge pull request #69 from ItsJuice/feat/improved-search

Feat/improved search

OCannings committed:
fix(Sockets): recursive dispatch on stream change

Fixing the tests revealed that an action was being dispatched
repeatedly when the streamId changed.

An action was being dispatched inside the subscribe callback before the
check which was designed to prevent recursion. (`newStream ===
oldStream`)

`subscribe.js` has been updated to ensure the action is only dispatched
once when the streamId changes.

Tests are now passing.
